lib/python/vdm/server/HTTPListener.py

```diff
@@ -438,8 +480,674 @@ def map_deployment_users(request, user):
     return deployment_user[0]
 
 
+def get_database_deployment(dbid):
```

Comment:
at a minimum breakout the deployment parsing/generation code into a separate module